src/sentry/static/sentry/app/views/settings/personal/personalNotifications.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import SettingsPageHeader from '../components/settingsPageHeader';
+import personalNotificationFields from '../../../data/forms/personalNotificationSettings';
+import Form from '../components/forms/form';
```

Comment:
You'll want `../components/forms/next/form` here

Reply:
ignoring this per our discussion on slack